### PR TITLE
marge the home branch to development branch

### PR DESCRIPTION
**Pull request by Riadul-Islam-551**
the home branch was created for understanding the conflict issue

### CODE REVIEW

File: src/styles/index.css
Comment (ginSpirit) [inline src/styles/index.css:1]: Your code is okay, it can approval

Comment (Riadul-Islam-551) [reply src/styles/index.css:1]: Thank you very much!!

Comment (ginSpirit) [inline src/styles/index.css:1]: It looks standard. nice work!

**Comment by Mahedi-432A** [comment]
ijfosiughiuoevhm iurfh dhgiusdvwiuehf aiygv uihgwiuhf ieywg fiui hgwsioufvhdfihvnlkj fnhoesiuvhwiuhsgfohrh gwe9hf iuweshf iuwh siudheoiugh iu heriuhsr gufbrgteroiusd vdiuhgifughuo iur hiuh giudghrefvciji uoug doiugh riueriu

**Review by ginSpirit — APPROVED** [review]
good job!

**Review by Md-Dipu — APPROVED** [review]
_**Thank you!**_

File: src/styles/index.css
Comment (Riadul-Islam-551) [reply src/styles/index.css:1]: thank u

**Pull request merged**